### PR TITLE
Fixing Launch Template Issue #768

### PR DESCRIPTION
**Pull request by elamaran11**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: examples/blueprint-construct/index.ts [hunk 1]
```diff
@@ -326,52 +327,20 @@ function addCustomNodeGroup(): blueprints.ManagedNodeGroup {
...
-      } Catch {
-        Throw $_
-      }
-    `);
-    const ebsDeviceProps: ec2.EbsDeviceProps = {
-        deleteOnTermination: false,
-        volumeType: ec2.EbsDeviceVolumeType.GP2
-    };
 
     return {
```
Comment (shapirov103) [inline examples/blueprint-construct/index.ts:331]: What prompted these changes? Are they intentional?

**Review by shapirov103 — COMMENTED** [review]
Looks great, please see my question.

File: examples/blueprint-construct/index.ts (hunk 1 above)
Comment (elamaran11) [reply examples/blueprint-construct/index.ts:331]: WIndows pattern prompted these changes. So we can use Disksize which takes care of EBS volume so we dont need to use LT.

**Comment by elamaran11** [comment]
@shapirov103 On checking the quick start account, this look like a VPC delete failure issue. Do you want to cleanup and rerun or we are good to merge?

**Pull request merged**